src/extra/flash/files/usr/bin/flash.lua

```diff
@@ -0,0 +1,52 @@
+local args, options = require("argutil").parse(...)
```

Comment:
this file should be in its own repository, maybe Wattana-Programs/flash.lua, then `setup` downloads that file to `files/usr/bin/setup.lua` - see comment on `setup` below